addons/im_livechat/models/discuss_channel.py

```diff
@@ -124,7 +124,7 @@ def _close_livechat_session(self, **kwargs):
                 return
             # Notify that the visitor has left the conversation
             self.message_post(author_id=self.env.ref('base.partner_root').id,
-                              body=self._get_visitor_leave_message(**kwargs), message_type='comment', subtype_xmlid='mail.mt_comment')
+                              body=self._get_visitor_leave_message(**kwargs), message_type='comment', subtype_xmlid='mail.mt_note')
```

Comment:
`mt_note` (and the use of `subtype_xmlid` in general) was I believe limited to chatter until now, and I'd prefer to keep it that way if possible.

Can't we use message type (notification?) instead?

Reply:
I don't think so, for the "Visitor left the conversation" it might be OK but it won't work for the rating + reason (would be very strange to have such a long notification)

Reply:
Would be indeed best to show "Visitor left the conversation" as a notification, with `message_type="notification"`.
For rating, we can turn this one into `mt_note`.

Reply:
Juliette told me she wanted it to be a note as well, so I think we could just keep it like it?